src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/Staff.java

```diff
@@ -3,20 +3,22 @@
 import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Data;
+import lombok.EqualsAndHashCode;
 import lombok.NoArgsConstructor;
 
 import javax.persistence.Column;
 import javax.persistence.Entity;
 import javax.persistence.Id;
 import javax.persistence.Table;
 
+@EqualsAndHashCode(of = "id", callSuper = true)
 @Entity
 @Table(name = "STAFF_MEMBERS")
-@Data()
+@Data
 @Builder
 @NoArgsConstructor
 @AllArgsConstructor
-public class Staff {
+public class Staff extends AuditableEntity {
```

Comment:
Do we create a Staff entity in the process?

Reply:
No, but just making consistant